### DISCUSSION TITLE
Outdated Wiki migration guide

### DISCUSSION THREAD

**Discussion by yordan-kanchelov** [pixijs/pixijs#9360] [General]
I recently started going through the release tags , alongside with the pixi.js migration guide. With the 7.2.0 release this paragraph now seems outdated

https://github.com/pixijs/pixijs/wiki/v7-Migration-Guide#-abandon-the-use-of-peerdependencies

**Comment by bigtimebuddy** [comment]
Updated this. Thanks for pointing it out
